bundle/manifests/logging.openshift.io_elasticsearches_crd.yaml

```diff
@@ -0,0 +1,370 @@
+                        properties:
+                          delete:
+                            nullable: true
+                            properties:
+                              minAge:
+                                description: The minimum age of an index before it
+                                  should be deleted (e.g. 10d)
+                                type: string
+                            type: object
+                          hot:
+                            nullable: true
+                            properties:
+                              actions:
+                                properties:
+                                  rollover:
+                                    nullable: true
+                                    properties:
+                                      maxAge:
+                                        description: The maximum age of an index before
+                                          it should be rolled over (e.g. 7d)
+                                        type: string
+                                    type: object
+                                type: object
+                            type: object
+                        type: object
+                      pollInterval:
+                        description: How often to check an index meets the desired
+                          criteria (e.g. 1m)
+                        type: string
+                    required:
+                    - phases
+                    - pollInterval
+                    type: object
+                  type: array
+              type: object
+            managementState:
+              description: Indicator if the resource is 'Managed' or 'Unmanaged' by
+                the operator
+              enum:
+              - Managed
+              - Unmanaged
+              type: string
+            nodeSpec:
+              description: Default specification applied to all Elasticsearch nodes
+              properties:
+                image:
+                  description: The image to use for the Elasticsearch nodes
+                  nullable: true
+                  type: string
+                nodeSelector:
+                  additionalProperties:
+                    type: string
+                  description: Define which Nodes the Pods are scheduled on.
+                  nullable: true
+                  type: object
+                proxyResources:
+                  description: The resource requirements for the Elasticsearch proxy
+                  nullable: true
+                  properties:
+                    limits:
+                      additionalProperties:
+                        anyOf:
+                        - type: integer
+                        - type: string
+                        pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                        x-kubernetes-int-or-string: true
```

Comment:
is this actually correct to have at this level? limits and requests are objects whose properties this would apply to (or does `additionalProperties` mean just this?)

Reply:
Actually `additionalProperties` mean just this. They are new shiny thing to express more complex validations as schema.